core/color.cpp

```diff
@@ -496,12 +496,12 @@ Color Color::operator*(const Color &p_color) const {
 			a * p_color.a);
 }
 
-Color Color::operator*(const real_t &rvalue) const {
+Color Color::operator*(real_t p_rvalue) const {
 	return Color(
-			r * rvalue,
-			g * rvalue,
-			b * rvalue,
-			a * rvalue);
+			r * p_rvalue,
+			g * p_rvalue,
+			b * p_rvalue,
+			a * p_rvalue);
```

Comment:
What is `rvalue` meant to represent here? Why not `p_value`?

Reply:
Assuming "right value", it's the same in 4.x (for `Vector2`, can change it but wanted to make as minimal changes as possible

Reply:
I mostly made it simple, and these names were used elsewhere as well, but will add to the cleanup :)

Reply:
`Vector2` for instance actually has a comedy amount of different namings for essentially just values of different types to be used as multipliers etc. :grinning: 

`p_v`
`rvalue`
`p_vec2`
`p_by`

I guess this is historical from the days where there wasn't much review and coding conventions. I personally don't get too worked up about it, but no doubt one day someone will do a super PR standardizing all this stuff I expect. :grin: